dockstore-webservice/src/main/java/io/dockstore/webservice/core/Doi.java

```diff
@@ -39,7 +39,7 @@
 
 @Entity
 @Schema(description = "A Digital Object Identifier (DOI)")
-@Table(name = "doi", uniqueConstraints = @UniqueConstraint(name = "unique_doi_name", columnNames = { "name" }))
+@Table(name = "doi", uniqueConstraints = { @UniqueConstraint(name = "unique_doi_name", columnNames = { "name" }), @UniqueConstraint(name = "unique_doi_initiator", columnNames = { "id", "initiator" }) })
```

Comment:
Need this unique constraint so that we can add (id, initiator) foreign keys in the join table